aiodogstatsd/contrib/aiohttp.py

```diff
@@ -3,7 +3,7 @@
 
 from aiohttp import web
 from aiohttp.web_app import _Middleware
-from aiohttp.web_routedef import _SimpleHandler
+from aiohttp.typedefs import Handler
```

Comment:
this will break compatibility with `aiohttp < 3.8.0`
should we use some kind of coniditional imports?

```
try:
  from aiohttp.typedefs import Handler
except ImportError:
  from aiohttp.web_routedef import _SimpleHandler as Handler
```